src/java/org/apache/cassandra/db/marshal/UserType.java

```diff
@@ -495,7 +495,7 @@ public String toCqlString(boolean withInternals, boolean ifNotExists)
                 builder.append(",")
                        .newLine();
 
-            builder.append(fieldNameAsString(i))
+            builder.appendQuotingIfNeeded(fieldNameAsString(i))
```

Comment:
actual bug, but only impacts toCQL so limited issue... too little for its own JIRA =(